rag/flow/parser/utils.py

```
@@ -170,9 +170,9 @@ def enhance_media_sections_with_vision(
 
     try:
         try:
-            vision_model_config = resolve_model_config(tenant_id, LLMType.IMAGE2TEXT, vlm_conf["llm_id"])
+            vision_model_config = resolve_model_config(tenant_id, LLMType.VISION, vlm_conf["llm_id"])
         except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)